xml/article_setup.xml

```diff
@@ -1103,7 +1106,7 @@ rmdir /dev/cpuset/$1
  </sect1>
  -->
   <sect1 xml:id="sec-quick-taskset">
-    <title>Specifying a CPU Affinity with <command>taskset</command></title>
+    <title>Specifying a CPU affinity with <command>taskset</command></title>
```

Comment:
```suggestion
    <title>Specifying CPU affinity with <command>taskset</command></title>
```

Reply:
I think the article is unnecessary here and makes a general statement more restrictive for no reason I can see. What do you think?

Reply:
I have no clue but tend to agree. It's mostly uncharted territory for me. Ultimately, I am not sure either.